libkbfs/mdserver_memory.go

```diff
@@ -88,6 +88,7 @@ type mdServerMemShared struct {
 	// tracks expire time and holder
 	lockIDs              map[mdLockMemKey]mdLockMemVal
 	implicitTeamsEnabled bool
+	iTeamMigrationLocks  map[tlf.ID]struct{}
```

Comment:
Slightly prefer a `map[tlf.ID]bool` instead so you can just `return md.iTeamMigrationLocks[id]` below.

Reply:
Hmmm OK!